lib/reline/line_editor.rb

```diff
@@ -1913,7 +1893,7 @@ def byte_pointer=(val)
     @cursor_max = calculate_width(@line)
   end
 
-  def whole_lines(index: @line_index, line: @line)
+  def whole_lines(index: @previous_line_index || @line_index, line: @line)
```

Comment:
Maybe this is better? because whole_lines is always called without arguments now
```ruby
def whole_lines
  index = @previous_line_index || @line_index
  line = @line
```